ra/ra.go

```diff
@@ -945,6 +956,22 @@ func (ra *RegistrationAuthorityImpl) validateFinalizeRequest(
 		return nil, berrors.BadCSRError("unable to parse CSR: %s", err.Error())
 	}
 
+	if ra.mustStapleAllowList != nil {
+		for _, ext := range csr.Extensions {
+			if !ext.Id.Equal(issuance.OCSPMustStapleExt.Id) {
```

Comment:
This loop can be replaced by `if issuance.ContainsMustStaple(csr.Extensions)`, and then issuance.OCSPMustStapleExt doesn't have to be exported.

Reply:
Ah, I hadn't noticed that method, thanks!